iOS/Flows/Conversation/Cells/MessageCell.swift

```diff
@@ -13,7 +13,11 @@ import SwiftUI
 class MessageCell: UICollectionViewCell {
 
     let content = MessageContentView()
-    let detailView = MessageDetailView()
+
+    // Detail View
+    @ObservedObject private var detailState = MessageDetailConfig(message: nil)
+    private lazy var detailView = MessageDetailView(config: self.detailState)
+    private lazy var detailVC = NavBarIgnoringHostingController(rootView: self.detailView)
```

Comment:
The detail state is basically an observable wrapper around a message. All the detail subviews have a reference to it so they can be updated when the message changes.